README.md

```diff
@@ -1 +1,14 @@
-# angular-filters-lab
\ No newline at end of file
+# Filters on datasets
+
+## Objectives
+
+- Pass a filter into the ng-repeat
+- Pass a filter into the ng-model's associated expression
+
+## Instructions
+
+In this repo we have a small app that displays a list of people and their birthdays.
```

Comment:
Add some fun context here: "as you can see, the list is really hard to sort through! let's add a filter so we can search for people"